### DISCUSSION TITLE
Images in archived albums and the main timeline on mac

### DISCUSSION THREAD

**Discussion by rollermy** [ente-io/ente#4000] [Q&A]
When adding an image to an archived album:

On my android, it removes it from the main timeline (per the [docs](https://help.ente.io/photos/features/archive)).

On the mac client,the image stays in the main timeline (all).

Is this a bug on the mac client?

Thanks!

**Comment by mnvr** [comment]
Changed in https://github.com/ente-io/ente/pull/4919
